ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+## Description
+
+Provide a description of your issue.
+For any suggestion, please address them on the [forum](https://forum.freegamedev.net/viewforum.php?f=16).
+For translations, go to [transifex](https://www.transifex.com/supertuxkart/supertuxkart/).
+
+## Steps to reproduce
+
+## Configuration
+
+STK release version:
+
+STK Source (ppa, distribution package, official bin,...):
+
+System: 
+
+Graphics card: 
+
+CPU: 
+
+Gamepads/keyboards models if related to the issue:
+
+## Additional informations
+
+Please provide stdout.log, it is located in
+
+* %appdata%\supertuxkart\0.8.2 (windows)
+* ~/.config/supertuxkart/0.8.2 (linux, mac and other unix based systems)
```

Comment:
This is invalid for OSX, maybe just point to "Where is the configuration stored?" on https://supertuxkart.net/FAQ

Reply:
There is no link to point directly at the right place in the page, I put the mac path in the file for now, if that is not wanted I'll rewrite it to simply point to the global FAQ.